services/harmony/app/util/parameter-parsers.ts

```diff
@@ -79,10 +79,25 @@ export function handleCrs(
  */
 export function handleScaleExtent(
   operation: DataOperation,
-  query: Record<string, string>): void {
+  query: Record<string, number[]>): void;
+export function handleScaleExtent(
+  operation: DataOperation,
+  query: Record<string, string>): void;
+export function handleScaleExtent(
+  operation: DataOperation,
+  query: Record<string, number[] | string>): void {
   if (query.scaleextent) {
-    const [xMin, yMin, xMax, yMax] = query.scaleextent;
-    operation.scaleExtent = { x: { min: xMin, max: xMax }, y: { min: yMin, max: yMax } };
+    if (typeof query.scaleextent === 'string') {
+      const scaleExtentString = query.scaleextent.replace('(', '').replace(')', '');
```

Comment:
The string being passed around here was e.g., `(-180, -90, 180, 90)`, so I wanted to strip off the outer brackets before splitting the string.

I did also consider `query.scaleextent.slice(1, -1)`, but that seemed a little less clear as to what I was doing.